lib/Websocket/Handshake.php

```diff
@@ -111,6 +111,13 @@ public function end(string $finalBodyChunk = ""): \Amp\Promise {
     /**
      * {@inheritDoc}
      */
+    public function abort(): \Amp\Promise {
```

Comment:
Why does this have `abort()`? And why does it declare `: Amp\Promise`?

Reply:
It has abort because interface. And the return type declaration is really too much.